indolaw-nextjs/components/Law.tsx

```diff
@@ -1,14 +1,14 @@
 import { Complex, renderChildren } from "utils/grammar";
```

Comment:
![](https://media2.giphy.com/media/VhWVAa7rUtT3xKX6Cd/giphy.gif)

Passing this back to you for the requested tweaks in the previous comments, ow overall looking good